app/src/main/java/com/devindi/wallpaper/model/map/TileRequestHandler.kt

```diff
@@ -7,6 +7,15 @@ import org.koin.standalone.KoinComponent
 import org.koin.standalone.inject
 import timber.log.Timber
 
+fun buildTileRequest(mapSourceId: String = "mapnik", width: Int = 100, height: Int = 100, zoom: Int = 0, centerLat: Double = 0.0, centerLon: Double = 0.0): String {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/devindi/mapwall/blob/c641b62fa39c60acdf91edee9483647734123681/app/src/main/java/com/devindi/wallpaper/model/map/TileRequestHandler.kt#L10"></span>Parameter should be on a separate line (unless all parameters can fit a single line)</td>
    </tr>
  </tbody>
</table>